src/app/user/modules/records/components/record-detail/record-detail.component.ts

```diff
@@ -34,6 +36,8 @@ export class RecordDetailComponent implements OnInit {
       this.record = data.record;
       this.isCampusEventRecord = Boolean(this.record.campus_event);
       this.hasFeedbackSent = this.record.status === RecordStatus.STATUS_FEEDBACK_SUBMITED;
+      this.couldEditStauts = this.record.status === RecordStatus.STATUS_SUBMITTED ||
```

Comment:
This comment is just a heads up, after @ghh82058 's PR been merged, you might need to update the `couldEdiStatus` to support more 'editable' statuses. BTW, `couldEditStauts` is a typo, please fix it, and I think `editable` is better than `couldEditStatus`, since the latter is more like a function name instead of a field name.